test/common/EdgeDevice.cs

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+namespace common
+{
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices;
+
+    public class EdgeDevice
+    {
+        readonly Device device;
+        readonly IotHub iotHub;
+        readonly bool owned;
+
+        public string ConnectionString =>
+            $"HostName={this.iotHub.Hostname};" +
+            $"DeviceId={this.device.Id};" +
+            $"SharedAccessKey={this.device.Authentication.SymmetricKey.PrimaryKey}";
+
+        public string Id => this.device.Id;
+
+        EdgeDevice(Device device, bool owned, IotHub iotHub)
```

Comment:
Make it private

Reply:
nit: Suggest to move IotHub to 1st argument.

Reply:
> Make it private

The constructor? It is private.

Reply:
> Make it private

yes, you already did.